src/components/Portal/__tests__/Portal.spec.js

```diff
@@ -3,31 +3,152 @@
  * @copyright IBM Security 2018
  */
 
-import { mount, shallow } from 'enzyme';
+import { cleanup, render, fireEvent } from '@testing-library/react';
 import React from 'react';
 
-import { className, label } from '../../_mocks_';
-
-import Portal from '../';
+import { Portal } from '../../..';
 
 describe('Portal', () => {
-  let portal;
-
-  beforeEach(() => {
-    portal = (
-      <Portal className={className} title={label}>
-        <div>
-          <button>TEST</button>
-        </div>
+  beforeEach(cleanup);
```

Comment:
I'm pretty sure that `cleanup` is usually [not necessary for React](https://testing-library.com/docs/react-testing-library/api#cleanup). Is it required here?

Reply:
Thank you, that makes sense! 👍 I have remove that